src/test/java/com/marklogic/kafka/connect/source/DslConstraintInjectionTest.java

```diff
@@ -1,61 +1,122 @@
-        Assertions.assertEquals(expectedValue, dslQueryHandler.injectConstraintIntoQuery(constraintValue));
+        String userDsl = "op.fromView('Medical', 'Authors').limit(1000)";
+        String expectedValue = "op.fromView('Medical','Authors').where(op.gt(op.col('ID'), '2')).limit(1000)";
+        assertEquals(expectedValue, injectValueIntoQuery(userDsl, parsedConfig, "2"));
     }
 
     @Test
     void testQueryWithLineFeed() {
-        String originalDsl = "op.fromView(\"Medical\", \"Authors\")  \n    .limit(1000)";
-        Map<String, Object> parsedConfig = new HashMap<String, Object>() {{
-            put(MarkLogicSourceConfig.DSL_QUERY, originalDsl);
-            put(MarkLogicSourceConfig.CONSTRAINT_COLUMN_NAME, constraintColumn);
-        }};
-        DslQueryHandler dslQueryHandler = new DslQueryHandler(null, parsedConfig);
-        String expectedValue = "op.fromView(\"Medical\",\"Authors\").where(op.gt(op.col(\"" + constraintColumn + "\"), \"" + constraintValue + "\")).limit(1000)";
-        Assertions.assertEquals(expectedValue, dslQueryHandler.injectConstraintIntoQuery(constraintValue));
+        String userDsl = "op.fromView('Medical', 'Authors')  \n    .limit(1000)";
+        String expectedValue = "op.fromView('Medical','Authors').where(op.gt(op.col('ID'), '2')).limit(1000)";
+        assertEquals(expectedValue, injectValueIntoQuery(userDsl, parsedConfig, "2"));
     }
 
     @Test
     void testQueryWithSpacesInSingleQuotes() {
-        String originalDsl = "op.fromView(\"Medical\", \"Authors\").select([op.as(\"Last Name\", op.col(\"LastName\"), op.as(\'Fore Name\', op.col(\'ForeName\')])";
-        Map<String, Object> parsedConfig = new HashMap<String, Object>() {{
-            put(MarkLogicSourceConfig.DSL_QUERY, originalDsl);
-            put(MarkLogicSourceConfig.CONSTRAINT_COLUMN_NAME, constraintColumn);
-        }};
-        DslQueryHandler dslQueryHandler = new DslQueryHandler(null, parsedConfig);
-        String expectedValue = "op.fromView(\"Medical\",\"Authors\").where(op.gt(op.col(\"" + constraintColumn + "\"), \"" +
-            constraintValue + "\")).select([op.as(\"Last Name\",op.col(\"LastName\"),op.as(\'Fore Name\',op.col(\'ForeName\')])";
-        Assertions.assertEquals(expectedValue, dslQueryHandler.injectConstraintIntoQuery(constraintValue));
+        String userDsl = "op.fromView('Medical', 'Authors').select([op.as('Last Name', op.col('LastName'), op.as('Fore Name', op.col('ForeName')])";
+        String expectedValue = "op.fromView('Medical','Authors').where(op.gt(op.col('ID'), '2')).select([op.as('Last Name',op.col('LastName'),op.as('Fore Name',op.col('ForeName')])";
+        assertEquals(expectedValue, injectValueIntoQuery(userDsl, parsedConfig, "2"));
+    }
+
+    @Test
+    void wordQueryWithPhraseOrderByAndLimit() {
+        String userDsl = "op.fromDocUris(cts.wordQuery('my phrase'))";
```

Comment:
For this test, I think it's worth ordering it by the "uri" column that comes back. I can see a user actually doing that (assuming they use URIs with a meaningful order to them), as ML does allow for ordering results based on URI order. And it's better to have a test where the constraint column exists in the rows coming back, even though we're not running the query in this test.